src/components/editor/property/InputLiteral.jsx

```diff
@@ -25,23 +25,16 @@ export class InputLiteral extends Component {
     this.state = {
       show: {},
       content_add: '',
-      disabled: false,
       lang_payload: null,
     }
     this.inputLiteralRef = React.createRef()
   }
 
-  componentDidMount = () => {
-    // Note that it is possible that the resource template hasn't been loaded yet, hence there will not be a property template.
-    if (this.props.propertyTemplate
-        && this.props.propertyTemplate.repeatable === 'false'
-        && this.props.formData !== undefined
-        && this.props.formData.items.length > 0) {
-      this.setState({ disabled: true })
-    }
+  disabled = () => {
+    return this.props.propertyTemplate.repeatable === 'false' &&
```

Comment:
@jcoyne I think we both worked on refactoring the same thing, but I see here that we do not need the componentDidMount method anymore... How about using this instead (import from Utilities) which uses safe navigation for `repeatable`?

`!booleanPropertyFromTemplate(this.props.propertyTemplate, 'repeatable', true)`

Reply:
This was already here, it's not something I changed in this PR. I try not to change more than is absolutely necessary. If you want to change it in another PR, please go ahead.

Reply:
Ok when this is out of draft I will merge it and then rebase.